src/lib.rs

```diff
@@ -133,6 +133,7 @@ pub mod test;
 
 #[cfg(test)]
 mod tests {
+    #[cfg(feature = "rsa_signing")]
```

Comment:
Note also that eventually (hopefully soon) we will make ECC (#247) and RSA verification work without using `BIGNUM`. Then, RSA signing will be the only thing using `BIGNUM` and we can indeed skip `bn_test.cc` when the RSA signing feature is not enabled. But, we don't need to add that skipping in *this* PR.